priv/build_schema.aql

```diff
@@ -0,0 +1,58 @@
+CREATE UPDATE-WINS TABLE FmkePatients (
```

Comment:
I know this wasn't done for the existing CQL file, but maybe it would make sense to have a 1-2 lines explaining what is AQL, maybe linking to the source or a documentation page. It would help distinguish what essentially are 2 almost identical files 🙂

Reply:
Done, I've added a link to the most recent AQL repository.